### PR TITLE
[DDW-870] Update the team member list

### PR DESCRIPTION
**Pull request by marcin-mazurek**
This PR updates the list of team members.

## Screenshots

Screenshot 2022-01-10 at 17 48 12
Screenshot 2022-01-10 at 17 48 23

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1641833458005200)
- [ ] Test

---

### Basics

- [x] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [x] PR has a good description that summarizes all changes
- [x] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [x] In English
  - [x] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [x] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [x] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [x] There are no *flow* errors or warnings (`yarn flow:test`)
- [x] There are no *lint* errors or warnings (`yarn lint`)
- [x] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [x] There are no missing translations (running `yarn manage:translations` produces no changes)
- [x] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [x] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [x] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [x] Important parts of the code are properly commented and documented
- [x] Code is properly typed with flow
- [x] React components are split-up enough to avoid unnecessary re-renderings
- [x] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/components/static/About.js [hunk 1]
```diff
@@ -30,13 +30,13 @@ const messages = defineMessages({
   aboutContentDaedalusMembers: {
     id: 'static.about.content.daedalus.members',
     defaultMessage:
-      '!!!Alan McNicholas, Aleksandar Djordjevic, Alexander Rukin, Brian McKenna, Charles Hoskinson, Danilo Prates, Darko Mijić, Dominik Guzei, Elin Liu, Gabriela Ponce, Jane Wild, Jeremy Wood, Juli Sudi, Junko Oda, Laurie Wang, Manus McCole, Michael Bishop, Mior Sufian, Nikola Glumac, Piotr Stachyra, Rhys Bartels-Waller, Richard Wild, Robert Moore, Rodney Lorrimar, Sam Jeston, Samuel Leathers, Serge Kosyrev, Tatyana Valkevych, Tomas Vrana, Tomislav Horaček, Yakov Karavelov',
+      '!!!Alan McNicholas, Aleksandar Djordjevic, Alexander Rukin, Amit Sagar, Brian McKenna, Charles Hoskinson, Daniel Main, Danilo Prates, Darko Mijić, Dmitrii Gaico, Dominik Guzei, Elin Liu, Gabriela Ponce, Jane Wild, Jeremy Wood, Juli Sudi, Junko Oda, Laurie Wang, Manus McCole, Marcin Mazurek, Michael Bishop, Michael Chappell, Mior Sufian, Nikola Glumac, Piotr Stachyra, Przemysław Włodek, Rhys Bartels-Waller, Richard Wild, Robert Moore, Rodney Lorrimar, Sam Jeston, Samuel Leathers, Serge Kosyrev, Szymon Masłowski, Tatyana Valkevych, Tomas Vrana, Tomislav Horacek, Tomislav Horaček, Yakov Karavelov',
```
Comment (mchappell) [inline source/renderer/app/components/static/About.js:33]: Duplication of Tomislav here, please remove one.

Comment (mchappell) [reply source/renderer/app/components/static/About.js:33]: Also missing (please add):
- Lucas Araujo
- Renan Ferreira

**Review by mchappell — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 - Updated `trezor-connect` dependency to version `8.2.4` ([PR 2726](https://github.com/input-output-hk/daedalus/pull/2726))
 - Updated vulnerable dependencies ([PR 2769](https://github.com/input-output-hk/daedalus/pull/2769))
 - Updated CWB and Cardano Node ([PR 2799](https://github.com/input-output-hk/daedalus/pull/2799))
+- Updated the list of team members ([PR 2805](https://github.com/input-output-hk/daedalus/pull/2805))
```
Comment (mchappell) [inline CHANGELOG.md:14]: Please move this to the top of the chores block and everything looks good to merge 👍

**Review by tomislavhoracek — APPROVED** [review]
@marcin-mazurek can you run `yarn manage:translations` please ?
Rest is 💯

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Tested on 20294. Great work @marcin-mazurek 👍

**Review by ManusMcCole — APPROVED** [review]

**Pull request merged**